compiler/gentree.cpp

```diff
@@ -27,6 +28,57 @@
 
 #define CE(x) if (!(x)) {return {};}
 
+// FindImplicitUses walks the arrow function body and finds variables
+// that need to be captured (by value)
+struct FindImplicitUses {
+  FindImplicitUses(VertexAdaptor<op_func_param_list> param_list, std::vector<VertexAdaptor<op_func_param>> *uses_of_lambda):
+    uses_of_lambda_{uses_of_lambda} {
+    skip_set_.reserve(param_list->params().size());
+    for (auto p : param_list->params()) {
+      skip_set_.insert(p.as<op_func_param>()->var()->str_val);
```

Comment:
I suggest to get var name via `get_string()` so we keep consistency with `visit_var()` method, or vice verse use str_val in  `visit_var()`.

Reply:
Done. Good suggestion.